Cogs.Tests/Cogs.Tests.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0-preview-20170427-09" />
+    <PackageReference Include="Newtonsoft.Json.Schema" Version="3.0.1" />

```

Comment:
We can't use the Newtonsoft.Json.Schema as it is GPL, and this project is MIT licensed. Lets remove it from the test project and think of another solution.